ddl-scripts/create-organization-tables.sql

```diff
@@ -0,0 +1,164 @@
+
+CREATE TABLE IF NOT EXISTS public.organization (
+    org_id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    parent uuid,
+    name VARCHAR(255),
+    status organizationstatus,
+    members uuid[],
+    owners uuid[],
+    FOREIGN KEY (parent) REFERENCES organization(org_id)
+);
+CREATE TYPE public.organizationstatus AS ENUM ('ORGANIZATION_STATUS_DRAFT', 'ORGANIZATION_STATUS_ACTIVE', 'ORGANIZATION_STATUS_SUSPENDED', 'ORGANIZATION_STATUS_RELEASED', 'ORGANIZATION_STATUS_TERMINATED');
+CREATE TABLE IF NOT EXISTS public.info (
+    info_id uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    org_id uuid NOT NULL,
+    name VARCHAR(255) NOT NULL,
+    short_name VARCHAR(255),
+    is_private BOOLEAN,
+    url VARCHAR(255),
+    logo VARCHAR(255),
+    tenant_id uuid,
+--    address_id uuid,
+--    FOREIGN KEY (address_id) REFERENCES address(address_id),
+--    FOREIGN KEY (tenant_id) REFERENCES tenant(tenant_id),
+    FOREIGN KEY (org_id) REFERENCES organization(org_id),
```

Comment:
This seems like it duplicates the `org_id` field on line 14, but I may just not be familiar with the syntax here